reporter/sources.py

```diff
@@ -355,6 +355,8 @@ class DBQueryErrorsSource(PHPLogsSource):
 * {backtrace}
 """
 
+    ER_LOCK_DEADLOCK = 1213
```

Comment:
Where does the 1213 come from? Maybe some comment with explanation would be useful.